website/static/js/conference.js

```diff
@@ -103,7 +103,6 @@ function Meeting(data) {
         filterStyle : { 'float' : 'right', 'width' : '50%'},
         allowMove : false,       // Turn moving on or off.
         hoverClass : 'fangorn-hover',
-        hiddenFilterRows : ['tags']
```

Comment:
Getting tags to filter properly would involve edits to treebeard so I decided just to stop filtering on tags.